tests/pytorch/nightly/resnet50-pod.libsonnet

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local common = import "common.libsonnet";
+local timeouts = import "templates/timeouts.libsonnet";
+local tpus = import "templates/tpus.libsonnet";
+local mixins = import "templates/mixins.libsonnet";
+
+{
+  local resnet50_pod = common.PyTorchGkePodTest {
+    modelName: "resnet50-mp",
+    command: [
+      "python3",
+      "/pytorch/xla/test/test_train_mp_imagenet.py",
+    ],
+  },
+  local functional = common.Functional {
+    command+: [
+      "--fake_data",
+    ],
+
+    workerCpu: "8",
+    workerMemory: "16Gi",
+  },
+  local convergence = common.Convergence {
+    command+: [
+      "--num_epochs=90",
+      "--datadir=/datasets/imagenet",
+      "--batch_size=128",
+      "--log_steps=200",
+    ],
+
+    workerCpu: "90",
```

Comment:
Are we reducing this per our discussion?

Reply:
Done. Lowered it to 36 CPU/100Gi, which is still above the maximum utilization that I'm seeing.